javascript/getDailyThreeHoursForecast.js

```diff
@@ -0,0 +1,24 @@
+import { displayDailyForecast } from './dailyForecast';
+import { displayThreeHourRange } from './threeHourRange';
+
+export const getDailyThreeHoursForecast = async () => {
+  // TODO Fix later to using param data
+  const lat = '49.2710362';
+  const lon = '-122.9808259';
+
+  const apiUrl = `https://api.openweathermap.org/data/2.5/forecast?lat=${lat}&lon=${lon}&appid=WEATHER_API_KEY&units=metric`;
+
+  try {
+    const res = await fetch(apiUrl);
+    const data = await res.json();
+
+    if (!res.ok) {
+      throw new Error(`HTTP error! Status: ${response.status}`);
+    }
+
+    displayDailyForecast(data);
+    displayThreeHourRange(data);
```

Comment:
Could you separate these functions from `getDailyThreeHoursForecast`?
I'm thinking the only responsibility for `getDailyThreeHoursForecast` should be to just fetch the weather from API, which should be something like this:
```javascript
const data = getDailyThreeHoursForecast();
displayDailyForecast(data);
displayThreeHourRange(data);
```

Reply:
That's very right! I changed it thanks!
https://github.com/Web2-Mid-Term-Project/weather_app/pull/7/commits/c19866d39da6152f6bc2a613534ced93833d482a